# Corresponds to the InterestPoint type in flirtlib
# Includes both the point location and optionally a descriptor

geometry_msgs/Pose2D pose

geometry_msgs/Point[] support_points

float32 scale

uint32 scale_level

DescriptorRos descriptor
